pdc_client/__init__.py

```diff
@@ -123,13 +123,18 @@ class to perform requests.
             if ca_cert_or_insecure is None:
                 ssl_verify = config.get(CONFIG_SSL_VERIFY_KEY_NAME)
                 insecure = config.get(CONFIG_INSECURE_KEY_NAME)
+                if ssl_verify == insecure:
+                    # Give a warning if ssl_verify == insecure in config file
+                    print("In config file, the values of ssl_verify and insecure can't be the same")
+                    sys.exit(1)
                 if insecure is not None:
                     sys.stderr.write("Warning: '%s' option is deprecated; please use '%s' instead\n" % (
                         CONFIG_INSECURE_KEY_NAME, CONFIG_SSL_VERIFY_KEY_NAME))
                     if insecure:
-                        ssl_verify = False
-                    elif not insecure and not ssl_verify:
-                        ssl_verify = True
+                        ssl_verify = not insecure
+                    else:
+                        # ssl_verify can be None here.
+                        ssl_verify = ssl_verify or not insecure
             elif ca_cert_or_insecure is True:
```

Comment:
Maybe add a little comments or/and use meaningful name to designate that ca_cert_or_insecure is from command line, can't be false; insecure is gotten from config file, etc, could make it easier to understand.